lib/apptentive_flutter.dart

```diff
@@ -169,6 +167,16 @@ class ApptentiveFlutter {
     return successful;
   }
 
+  static Future<int> getUnreadMessageCount() async {
```

Comment:
Right now async/await is only available for iOS 15 and later, although once Xcode 13.2 comes out of beta it will work back to iOS 13. Do you happen to know how far back this plugin will be expected to run (in terms of iOS version)?